react/FilePath/Readme.md

```diff
@@ -0,0 +1,21 @@
+#### File Path inside a non interactive container
+
+```
+import FilePath from 'cozy-ui/transpiled/react/FilePath';
+
+<FilePath text="/Partages reçus/Cozy 🗄 - Team/Customers & Partners 🛒/Xxxxxx (Xxxxxxxx)/4_Suivi opérationnel/Point de synchro" />
+```
+
+#### File Path inside an interactive container
+
+```
+import FilePath from 'cozy-ui/transpiled/react/FilePath';
+
+<a href="#"><FilePath text="/Partages reçus/Cozy 🗄 - Team/Customers & Partners 🛒/Xxxxxx (Xxxxxxxx)/4_Suivi opérationnel/Point de synchro" /></a>
+```
+
+```
+import FilePath from 'cozy-ui/transpiled/react/FilePath';
+
+<a href="#" className="u-inverted-link"><FilePath text="/Partages reçus/Cozy 🗄 - Team/Customers & Partners 🛒/Xxxxxx (Xxxxxxxx)/4_Suivi opérationnel/Point de synchro" /></a>
+```
```

Comment:
you should definitly use Link: https://docs.cozy.io/cozy-ui/react/#!/Link

https://material-ui.com/api/link/#css

Reply:
u-inverted-link doesn't exist anymore